.github/actions/f3d-superbuild/action.yml

```diff
@@ -42,18 +42,20 @@ runs:
     with:
       path: ${{env.SCCACHE_CACHE}}
       key: sccache-cache-${{inputs.raytracing_label}}-${{runner.os}}-0-${{env.DATE_STRING}}
-      restore-keys: sccache-cache-${{inputs.raytracing_label}}-${{runner.os}}-1
+      restore-keys: sccache-cache-${{inputs.raytracing_label}}-${{runner.os}}-2
```

Comment:
no need to change the cache number,

Reply:
and I just realised but this should be a zero, not a 1, to be coherent. It is still working as intended.

Reply:
I changed the MSVC version used, I think it's necessary to invalidate the cache, otherwise I have link errors.

Reply:
you can now update the superbuild instead